Resources/public/js/fragmentList.jquery.js

```diff
@@ -0,0 +1,337 @@
+            }
+
+            // Show errors ?
+            if(hasErrors) {
+                $fragment.addClass(this.options.errorState);
+            }
+
+            // Is it the first item OR force active state
+            if(!this.$activeFragment || isNew) this.setActive($fragment);
+
+            // Append to the DOM
+            this.$list.append($fragment);
+        },
+
+        /**
+         * Reset order after and item is dropped or removed
+         */
+        resetOrder: function() {
+            var self = this;
+
+            this.$list.children('li').each(function(index) {
+                var $this = $(this),
+                    $form = self.getFormByFragmentId($this.data('fragId')),
+                    $positionField = $form.find(self.options.formPositionName);
+
+                $positionField.val(index);
+            });
+        },
+
+        /**
+         * Check validity for a fragment (HTML5 validation only)
+         */
+        checkValidity: function() {
+            var isValid = true,
+                $form;
+
+            // Reset validity
+            if(this.$invalidFields) this.setValid(this.$invalidFields, true);
+
+            // Currently no fragments, current fragment is valid
+            if(!!this.$activeFragment) {
+                // Check current visible form
+                $form = this.getFormByFragmentId(this.$activeFragment.data('fragId'));
+                // Get invalid fields for this form
+                this.$invalidFields = $form.find('input:invalid, textarea:invalid');
+                // Is the fragment valid ?
+                isValid = !this.$invalidFields.length;
+                // Show invalid fields
+                if(!isValid) this.setValid(this.$invalidFields, false);
+            }
+
+            return isValid;
+        },
+
+        /**
+         * Set a field valid / invalid into a form
+         */
+        setValid: function($fields, isValid) {
+            var $group = $fields.closest('.form-group');
+
+            if(!isValid) {
+                $group.addClass('has-error');
+                this.scrollTo($group);
+            } else {
+                $group.removeClass('has-error');
+            }
+        },
+
+        /**
+         * Scroll to a DOM element
+         */
+        scrollTo: function($element) {
+            var offset = this.options.pageScrollOffset;
+
+            $('html, body').animate({
+                scrollTop: $element.offset().top - offset
+            }, this.options.pageScrollDuration);
+        }
+    };
+
+    // A really lightweight plugin wrapper around the constructor,
+    // preventing against multiple instantiations
+    $.fn[pluginName] = function ( options ) {
+        return this.each(function () {
+            if (!$.data(this, 'plugin_' + pluginName)) {
+                $.data(this, 'plugin_' + pluginName, new Plugin(this, options));
+            }
+        });
+    };
+})( jQuery, window, document );
```

Comment:
No space at begin and end of argument list.

Nice one but can this be applied here too? https://github.com/sonata-project/SonataArticleBundle/pull/2/files#diff-fc729806308bfd7dc9b7106ae4b55e16R1